### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by jameslnewell**
Had to bump the major version for moving the `font-face` into a mixin (so we don't get it duplicated all over the place) so I thought I'd implement a bunch of fixes while I was at it

### CODE REVIEW

**Comment by lauriejones** [comment]
Cool, looks good. Does moving the font-face to a mixin change how we use it in projects?

**Comment by jameslnewell** [comment]
yeah, we'll need to `@include icon-font-face()` in the global CSS file, import the compiled version of icons where we want the classes and continue importing icons as normal everywhere else where we're just using the mixins

**Comment by lauriejones** [comment]
Cool, I think we're probably using the compiled classes more often than mixins, but that doesn't matter.

**Comment by jameslnewell** [comment]
Yeah, and we should only import compiled classes once in the global css

**Comment by jameslnewell** [comment]
Hey @scrumtech, as this is a major breaking change, it'd be awesome to get a second set of 👀  for approval.

This PR does the following:

- exports the mixins and not the classes by default (so we don't repeat them multiple times)
- moves the font face to a mixin

Which requires the following changes to apps (and the odd component which I'll change):
- `@import "@nib-styles/v2-icons/dist/compiled";` in your top level stylesheet
- leave the existing `@import "@nib-styles/v2-icons";` to import mixins in your other stylesheets

It appears to fix issues with code splitting in Chrome where the `@font-face` listed multiple times appears to prevent it from showing.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nib-styles/v2-icons",
-  "version": "2.12.1",
-  "main.css": "dist/compiled.scss",
+  "version": "3.0.0-preview.2",
```
Comment (scrumtech) [inline package.json]: Suggest ditching the preview.2 bit

Comment (jameslnewell) [inline package.json]: 👍

**Pull request merged**